libs/apigateway-lambda-proxy/src/lib/metric-alarms.ts

```diff
@@ -0,0 +1,376 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as aws from '@pulumi/aws'
+
+const namespace = 'AWS/ApiGateway'
+
+/**
+ * Dimensions to identify an HTTP API in CloudWatch.
+ *
+ * The main difference is REST APIs are identified by name, while HTTP APIs are identified by id.
+ */
+export type HttpGateway = {
+    /**
+     * The API Gateway ID to monitor. Only valid for HTTP endpoints.
+     */
+    id: pulumi.Input<string>
+
+    /**
+     * The API name to monitor.
+     *
+     * For REST endpoints, this must be the name of the API created in AWS.
+     * For HTTP endpoints, this can be anything (it's only used in alerts).
+     */
+    name: pulumi.Input<string> | undefined
```

Comment:
Can be cleaned up

Reply:
This is still used in the alarmDescription. Otherwise the alerts will just show the ApiId, which may not be super-helpful.

Reply:
Ah k, the docs on the prop can be cleaned up then. We should also make it mandatory?